tests/layer_tests/tensorflow_tests/test_tf_LogicalOps.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing Logical operations (Initial Implementation)
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/All
+#                https://www.tensorflow.org/api_docs/python/tf/raw_ops/Any
+
+class TestLogicalOps(CommonTFLayerTest):
+    # Overload inputs generation to fill dummy input
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.random.randint(0, 2, inputs_dict[input]).astype(bool)
+        return inputs_dict
+
+    # input_shape - should be an array
+    # axis - array which points on axis for the operation
+    # op_type - type of tested operation
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_logical_ops_placeholder_const_net(self, input_shape, axis, op_type, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->op_type   =>       Placeholder->ReduceLogicalAnd/Or
+                         /                               /
+            Const-------/                   Const-------/
+
+        """
+
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            op_type_to_tf = {
+                'All': tf.raw_ops.All,
+                'Any': tf.raw_ops.Any,
+            }
+            tf_input_shape = input_shape.copy()
```

Comment:
please do not create a copy of the shape, it is not needed, remove this in the next PR.